### DISCUSSION TITLE
Enrichment in questionnaires

### DISCUSSION THREAD

**Discussion by AleVirtanen** [helsenorge/refero#158] [Q&A] [answered]
I saw that in structor there is this enrichment option for inputfields using this extension: "http://ehelse.no/fhir/StructureDefinition/sdf-fhirpath". How have you implemented that extensions functionality and how it is integraded in components functionality?

**Comment by kennethmyhra** [comment, marked as answer]
Enrihment is implemented in our backend, therefore there is unfortunately no support for this in refero.
